mantidproject/mantidimaging
# Add environment setup

Use Conda environments for CI and as a simpler install method.

To test:
- [Create an environment](https://conda.io/docs/user-guide/tasks/manage-environments.html#creating-an-environment-from-an-environment-yml-file) from once of the two `.yml` files (one is Python 2.7, the other is 3.5)
- Activate the environment and install mantidimaging
- See that the GUI launches and all filters appear

(also fixes #207)
